README.md

```diff
@@ -1,2 +1,51 @@
-# cover-router
+# Cover Router
 Computes optimal capacity allocation per Nexus Mutual staking pool on cover purchase
```

Comment:
maybe rephrase to: 
`Computes the optimal route for capacity allocation in order to get the best price on cover purchases.`